orchagent/portsorch.cpp

```diff
@@ -3909,7 +4051,7 @@ bool PortsOrch::addLagMember(Port &lag, Port &port, bool enableForwarding)
     attr.value.oid = port.m_port_id;
     attrs.push_back(attr);
 
-    if (!enableForwarding)
+    if (!enableForwarding && port.m_type != Port::SYSTEM)
```

Comment:
Does this mean we cannot disable forwarding for sys lag?

Reply:
Setting (enable or disable) the ingress and egress is not applicable for remote LAG members. SAI returns errors when we set these attributes for system port lag members These are done at the owner ASIC (as local physical port members)